extensions/adobe/experience/offer-management/eligibility-rule.schema.json

```diff
@@ -0,0 +1,42 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$id": "https://ns.adobe.com/experience/offer-management/eligibility-rule",
+  "meta:extensible": "true",
+  "type": "object",
+  "title": "Eligibility Rule",
+  "description": "Eligibility rule encapsulates a profile query",
+  "definitions": {
+    "eligibility-rule": {
+      "properties": {
+        "@id": {
+          "type": "string",
+          "format": "uri",
+          "title": "ID",
+          "description": "The unique identifier of the filter"
+        },
+        "xdm:name": {
+          "type": "string",
+          "title": "Name",
+          "description": "Filter name"
+        },
+        "xdm:value": {
+          "type": "string",
+          "title": "Expression",
+          "description": "Profile Query"
```

Comment:
This is another use case for #362

Reply:
Looking at #362 that one seems to aim at a constant language. The expression here is a different query language that is used for offer decisioning and profile segmentation. Very different use cases indeed.

Reply:
Here's my thinking around this (without having seen the query language you plan to use):

For an eligibility rule, you want to look at a given customer profile and then decide if the customer is eligible for the offer. Therefore you compare the properties of the customer profile object to a set of conditions. 

In #362 you do the same thing:
- compare a vistor profile against conditions to determine if they should see a certain price
- compare a profile against conditions to determine if they should be considered part of an audience, etc.

Reply:
@jbeckert please comment

Reply:
We are talking to various PMs to see if there is even a use case to unify just eligibility constraints for offers and segmentation expressions for profiles.
On a very abstract level, I can see your point for a super-general XDM Query Expression format. However, it is too early to act on that desire/idea. This might not materialize at all.
I am strongly advocating to separate out that discussion from this PR and continue it on #362 
The way #362 is written it aims at a different use case on top of a general XDM instance matching/query language.